Mathlib/FieldTheory/IntermediateField.lean

```diff
@@ -611,6 +609,12 @@ theorem toSubalgebra_injective :
   rw [← mem_toSubalgebra, ← mem_toSubalgebra, h]
 #align intermediate_field.to_subalgebra_injective IntermediateField.toSubalgebra_injective
 
+theorem map_injective {f : L →ₐ[K] L'} (hf : Function.Injective f) :
+    Function.Injective (map f) := by
```

Comment:
You should be able to use [RingHom.injective](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Field/Basic.html#RingHom.injective) to remove `hf`

Reply:
Yes. you're right. Thanks